src/Cardano/Wallet/Api/Server.hs

```diff
@@ -0,0 +1,187 @@
+{-# LANGUAGE DuplicateRecordFields #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE OverloadedLabels #-}
+
+module Cardano.Wallet.Api.Server
+    ( server
+    ) where
+
+-- |
+-- Copyright: © 2018-2019 IOHK
+-- License: MIT
+--
+-- API handlers and server using the underlying wallet layer to provide
+-- endpoints reachable through HTTP.
+
+import Prelude
```

Comment:
how this can not compile ?

Reply:
:thinking:  Good question? Probably `haddock` that messes up things.